low-level/files/opt/seagate/sspl/bin/conf_based_sensors_enable

```diff
@@ -1,60 +1,85 @@
-    # in sspl_test.
-    sed_expr+=(-e '/^monitor=.*$/{ x ; /^\['"${group}"'\]$/{ x ; s/^monitor=.*$/monitor='"${monitor[${group}]}"'/;  b} ; x }')
-done
-
-
-[ -f "${SSPL_CONF}" ] && {
-    sed -i "${sed_expr[@]}" $SSPL_CONF
-}
-for group in "${!monitor[@]}"; do
-    ${CONSUL_PATH}/consul kv put sspl/config/"${group}"/monitor "${monitor["${group}"]}"
-done
-
-file_store_config_path="/opt/seagate/${PRODUCT_FAMILY}/sspl/sspl_test/conf/sspl_tests.conf"
-[ -f "${file_store_config_path}" ] && {
-    sed -i "${sed_expr[@]}" $file_store_config_path
-}
+#!/usr/bin/python3.6
 
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+import configparser
+import sys
+import os
+
+# Add the top level directories
+sys.path.insert(0, f'/opt/seagate/cortx/sspl/low-level')
+from framework.base.sspl_constants import (CONSUL_HOST, CONSUL_PORT, PRODUCT_NAME,
+    PRODUCT_FAMILY, file_store_config_path, storage_type, server_type, SSPL_STORE_TYPE)
+
+# TODO: Update sensor information in ConfStore
+# from cortx.utils.conf_store import Conf
+
+
+config = configparser.ConfigParser(allow_no_value=True)
+config.read(file_store_config_path)
+
+key = 'monitor'
+
+sensors = dict()
+sensors["REALSTORSENSORS"] = "true"
+sensors["NODEHWSENSOR"] = "true"
+sensors["SYSTEMDWATCHDOG"] = "true"
+sensors["RAIDSENSOR"] = "true"
+sensors["SASPORTSENSOR"] = "true"
+sensors["MEMFAULTSENSOR"] = "true"
+sensors["CPUFAULTSENSOR"] = "true"
+
+if storage_type.lower() in ["virtual", "jbod"]:
+    sensors["REALSTORSENSORS"] = "false"
+
+if server_type.lower() in ["virtual"]:
+    sensors["NODEHWSENSOR"] = "false"
+    sensors["SASPORTSENSOR"] = "false"
+    sensors["MEMFAULTSENSOR"] = "false"
+    sensors["CPUFAULTSENSOR"] = "false"
+    sensors["RAIDSENSOR"] = "false"
+
+for sect, value in sensors.items():
+    config.set(sect, key, value)
+
+if os.path.exists(file_store_config_path):
+    with open(file_store_config_path, "w") as configFile:
+        config.write(configFile, space_around_delimiters=False)
+
+# Onward LDR_R2, consul will be abstracted out and it won't exit as hard dependeny of SSPL
+if SSPL_STORE_TYPE == "consul":
```

Comment:
Add Note: Keeping for backward compatibility with LDR R1

Reply:
This will be addressed in EOS-16669